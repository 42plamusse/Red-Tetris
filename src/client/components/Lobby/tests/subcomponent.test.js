import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import LobbySub from "../subcomponent";

describe("<LobbySub>", () => {
  test("should render without crashing", () => {
    const wrapper = shallow(<LobbySub />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
