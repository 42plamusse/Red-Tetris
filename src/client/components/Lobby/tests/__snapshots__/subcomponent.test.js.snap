// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LobbySub> should render without crashing 1`] = `
<styled.div>
  <styled.div>
    <Connect(CreateRoomSub) />
    <Connect(RoomListSub) />
  </styled.div>
  <styled.div>
    <Connect(LadderSub) />
    <Connect(ChatSub) />
  </styled.div>
</styled.div>
`;
